microapps/ArWorldScaleApp/app/src/main/java/com/arcgismaps/toolkit/arworldscaleapp/screens/MainScreen.kt

```diff
@@ -77,25 +75,35 @@ import com.arcgismaps.mapping.Basemap
 import com.arcgismaps.mapping.BasemapStyle
 import com.arcgismaps.mapping.ElevationSource
 import com.arcgismaps.mapping.layers.ArcGISSceneLayer
+import com.arcgismaps.mapping.symbology.SceneSymbolAnchorPosition
 import com.arcgismaps.mapping.symbology.SimpleMarkerSceneSymbol
 import com.arcgismaps.mapping.symbology.SimpleMarkerSceneSymbolStyle
-import com.arcgismaps.mapping.symbology.Symbol
-import com.arcgismaps.mapping.symbology.SymbolStyle
 import com.arcgismaps.mapping.view.Graphic
 import com.arcgismaps.mapping.view.GraphicsOverlay
+import com.arcgismaps.mapping.view.SurfacePlacement
 import com.arcgismaps.toolkit.ar.WorldScaleSceneView
-import com.arcgismaps.toolkit.ar.WorldScaleSceneViewProxy
 import com.arcgismaps.toolkit.ar.WorldScaleSceneViewStatus
 import com.arcgismaps.toolkit.ar.WorldScaleTrackingMode
 import com.arcgismaps.toolkit.ar.rememberWorldScaleSceneViewStatus
 import com.arcgismaps.toolkit.arworldscaleapp.R
 
 private const val KEY_PREF_ACCEPTED_PRIVACY_INFO = "ACCEPTED_PRIVACY_INFO"
 
+val graphicSymbol = SimpleMarkerSceneSymbol(
+    style = SimpleMarkerSceneSymbolStyle.Cone,
+    color = Color.red,
+    height = 0.5,
+    width = 0.5,
+    depth = 0.5,
+    anchorPosition = SceneSymbolAnchorPosition.Bottom
+).apply {
```

Comment:
I propose making the cone slightly transparent and a little smaller. Also renaming the variable to `coneSymbol` which is more descriptive:
```suggestion
val coneSymbol = SimpleMarkerSceneSymbol(
    style = SimpleMarkerSceneSymbolStyle.Cone,
    color = Color.fromRgba(255, 0, 0, 70),
    height = 0.4,
    width = 0.2,
    depth = 0.2,
    anchorPosition = SceneSymbolAnchorPosition.Bottom
).apply {
```